### PR TITLE
Update peach-config documentation

### PR DESCRIPTION
**Pull request by mhfowler**
As I started working on this, I realized the setup docs could use a broader re-write, to look at how we want to easily communicate to users the recommended way to install, vs other possibilities for installation. But maybe that's a separate issue, and can start with this basic re-write for now, so that this repo doesn't have wrong information.

### CODE REVIEW

**Comment by mhfowler** [comment]
Another way to say this:

if peach-config is a module which handles installation and updating, and possibly even communicates with other peach microservices, that documentation should probably live in the README here, about this module, and how to use it.  

And the general documentation for users for setting up PeachCloud (which could refer to this module, but for most users is probably not making use of this module directly), should it live here or should it live somewhere else? Maybe it could just live in the "setup" section of docs.peachcloud.org? This could also reduce duplicate documentation. 

We could have the READMEs lean towards just documenting what a microservice is, and how to use it once its installed. 
And then more general documentation could live in docs.peachcloud.org? What do you think?

**Comment by mycognosist** [comment]
@mhfowler 

> We could have the READMEs lean towards just documenting what a microservice is, and how to use it once its installed. And then more general documentation could live in docs.peachcloud.org? What do you think?

This sounds desirable to me. The README for each repo acts as a self-contained document, while the developer docs are more geared towards providing an overview of the system: the design philosophy, architecture notes, how the parts fit together and how to setup and update the system.

It's probably also worthwhile to mention that the content of the `docs.peachcloud.org` site is aimed at developers, and that a separate document will be created for Peach caretakers / deployers / operators. So there's three levels of documentation going from most-detailed and low-level (README) to most-accessible and high-level (PeachCloud Guidebook), with the Developer Documentation in the middle.

Do you think that distinction and structuring makes sense?

**Comment by mhfowler** [comment]
@mycognosist the three levels sounds great, good to know, and gives me a more clear sense of for who and why I am writing documentation in each place. Maybe we can include the three levels in the developer documentation, if its not there already: 
READMES (most low level, APIs) 
docs.peachcloud.org (for developers) 
PeachCloud Guidebook (for peach caretakers /deployers / operators)

Also like the phrase peach caretakers / deployers / operators. 

So I will remove docs from here that are not directly about how to use peach-config as a module, 
and move those to docs.peachcloud.org

File: README.md [hunk 1]
```diff
@@ -1,111 +1,59 @@
...
-```
-
-[ Add the following two lines to top of file ]
+You can find a guide for installing plain Debian onto a Raspberry pi [here](docs/installation-from-debian-disc-image.md).
 
-```plaintext
-ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
-update_config=1
-```
+One you have Debian running on your pi, you can install peach-config by adding the PeachCloud apt repository and using apt.
```
Comment (mycognosist) [inline README.md:19]: Small typo here: 'One' -> 'Once'

File: docs/installation-from-debian-disc-image.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
+## Installation From Debian Disc Image
+
+You can also setup PeachCloud by installing Debian onto an sd card, and then installing and running peach-config. 
+This is essentially how the PeachCloud disc image is created (see [peach-img-builder](link)).
```
Comment (mycognosist) [inline docs/installation-from-debian-disc-image.md:4]: Don't forget the link :)

https://github.com/peachcloud/peach-img-builder

**Review by mycognosist — APPROVED** [review]
This documentation is a thing of beauty :) I really appreciate all the care and detail you've poured into this.

Looks like some of the `docs/` are still a work in progress. I like the `TODO` sections you've outlined. You're welcome to merge at your leisure.

**Pull request merged**